src/main/java/com/blamejared/recipestages/proxy/ClientProxy.java

```diff
@@ -42,7 +43,7 @@ public void syncJEI(EntityPlayer player) {
             Recipes.recipes.values().forEach(list -> {
                 for(IRecipe recipe : list) {
                     IRecipeWrapper recipeWrapper = recipeRegistry.getRecipeWrapper(recipe, VanillaRecipeCategoryUid.CRAFTING);
-                    if(recipeWrapper != null) {
+                    if(recipeWrapper != null && Configurations.showRecipe == false) {
```

Comment:
Wouldn't it be better to wrap the whole `Recipes.recipes.values().forEach` inside of:
```java
if(!Configurations.showRecipe){
Recipes.recipes.values().forEach(
```
that way it won't loop the list if the config is false

Reply:
Oh, right, sure